notebooks/5090_download-scenarios.py

```diff
@@ -190,8 +195,17 @@ def check_negatives(df):  # noqa : D103
         )
 
         msg = f"Negative values found in rows with indices:\n{negative}"
+        warnings.warn(msg)
 
-        raise AssertionError(msg)
+        for idx, row in tmp_not_co2[negative_rows].iterrows():
+            neg_rows = row.where(row < -(10**-6)).dropna()
+
+            if not neg_rows.empty:
+                err = [(idx, col, val) for col, val in zip(neg_rows.index, neg_rows.tolist())]
+                msg = f"Negative values found:\n{err}"
+                raise ValueError(msg)
+
+        df_ts.clip(lower=0)
```

Comment:
We have to be more careful about this no? Won't this affect every row (including CO2), not just the row of interest? (Or it affects nothing because we're not assigning the output?)